src/lib.rs

```diff
@@ -45,6 +45,14 @@ pub extern crate serde;
 
 use core::{fmt, str};
 
+// note: re-exports from `rand` crate shall be used. Otherwise trait bounds in `Mnemonic::generate_in_with` can get
+// unsatisfied if crate deps get ouf of sync.
+#[cfg(all(feature = "rand", feature = "rand_core"))]
+use rand::{CryptoRng, RngCore};
+
```

Comment:
If you like my comment suggestion then I'd remove the whitespace so it covers all 4 lines.